cli/resources.rs

```diff
@@ -186,7 +197,8 @@ impl Resource {
           "Another accept task is ongoing",
         ));
       }
-      t.replace(futures::task::current());
+      // TODO(afinch7) replace this.
+      // t.replace(futures::task::current());
```

Comment:
We need `Waker` for this. See https://github.com/tokio-rs/tokio/pull/1224. The trick is to store context pointer somewhere and create Waker from it.

Reply:
`AtomicWaker` ?

Reply:
`std::task::Waker`